ANMN/temperature_grid_product/anmn_temp_gridded_product.py

```diff
@@ -63,8 +63,8 @@ def get_good_values(var, var_qc):
     """
     Return the variable values which qc flag is 0, 1 or 2
     """
-    var = var[var_qc <= 2]
-        
+    var = [i for i,j in zip(var, var_qc) if j <= 2]
+    
```

Comment:
i actually don't care at all, but this extra empty line is not needed (see PEP8)

Reply:
Thank @lbesnard , if happy then could you please merge?